File: Flutter App/unicoVehicle/lib/Models/VehicleReview.dart
import './User.dart';
import './Miscellaneous/Vehicle.dart';

class VehicleReview {
  final int vehicleReviewId;
  final String review;
  final User user;
  final Vehicle vehicle;

  VehicleReview({
    this.review,
    this.user,
    this.vehicle,
    this.vehicleReviewId,
  });
}

File: UnicoVehicle/UnicoVehicle/UnicoVehicle.DTO/VehicleReview.cs
using System;
namespace UnicoVehicle.DTO
{
    public class VehicleReview
    {
        public int VehicleReviewId { get; set; }
        public User User { get; set; }
        public Miscellaneous.Vehicle Vehicle { get; set; }
        public string Review { get; set; }
    }
}
